ST_mortality.c

```diff
@@ -843,25 +945,25 @@ static void _stretched_clonal( GrpIndex rg, Int start, Int last,
   Mem_Free(clist);
 }
 
-/***********************************************************/
+/**
+ * \brief Kill all annuals.
```

Comment:
change to:

"Kill all individuals belonging to annual species"

Reply:
Addressed by commit 984bd13.